saleor/extensions/plugins/avatax/__init__.py

```diff
@@ -190,7 +190,7 @@ def append_line_to_data(
             "taxCode": tax_code,
             "taxIncluded": tax_included,
             "itemCode": item_code,
-            "description": description[:2000] if description else "",
+            "description": name,
```

Comment:
Is `description` nullable?

Reply:
should be `name` and yes I think it can be None

Reply:
It is. Maybe we could add a comment linking to the docs? https://developer.avalara.com/api-reference/avatax/rest/v2/models/LineItemModel/

Reply:
It is. Maybe we could add a comment linking to the docs? https://developer.avalara.com/api-reference/avatax/rest/v2/models/LineItemModel/

Reply:
I don't think it's necessary, it's not a bug, or missing feature, if we put documentation link here, why not add documentation link everywhere?